pkg/engine/lifecycletest/import_test.go

```diff
@@ -403,17 +403,36 @@ func TestImportUpdatedID(t *testing.T) {
 	p.Options.Host = deploytest.NewPluginHost(nil, nil, program, loaders...)
 
 	p.Steps = []TestStep{{Op: Refresh, SkipPreview: true}}
-	snap := p.Run(t, nil)
+
+	// Refresh requires at least one resource in order to proceed.
+	stackURN := resource.URN("urn:pulumi:stack::stack::pulumi:pulumi:Stack::foo")
+	stackResource := newResource(
+		stackURN,
+		"",
+		"foo",
+		"",
+		nil,
+		nil,
+		nil,
+		false,
+	)
+	snap := p.Run(t, &deploy.Snapshot{Resources: []*resource.State{stackResource}})
+
+	require.NotEmpty(t, snap.Resources)
```

Comment:
BTW this test had a bug where it wasn't actually performing a refresh (or running any subsequent asserts) due to having 0 resources. It might be worth checking if other tests are affected!